### PR TITLE
Implement Widget

### PR DESCRIPTION
**Pull request by donnapep**
* Use rise-storage web component to get files (storage.js)
* Use slider-revolution for slideshow functionality (slider.js)

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Please review. Thx.

File: src/widget/storage.js [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+   *  Private Methods
+   */
+  function init() {
+    var storage = document.querySelector("rise-storage"),
+      sort = "",
+      sortDirection = "";
+
+    storage.addEventListener("rise-storage-response", function(e) {
+      slider = new RiseVision.ImageFolder.Slider(params);
+      slider.init(e.detail);
```
Comment (stulees) [inline src/widget/storage.js]: @donnapep I understand you need the `e.detail` for initializing the slider, but (just my 2 cents) I feel like the `slider` instance should be created and maintained in `ImageFolder`. Creating it here in `ImageFolder.Storage` wouldn't be where I would expect to find it.

Comment (donnapep) [inline src/widget/storage.js]: @stulees This is fixed. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**